src/amo/components/SearchFilters/index.js

```diff
@@ -100,6 +103,7 @@ export class SearchFiltersBase extends React.Component {
     return [
       { children: i18n.gettext('All'), value: NO_FILTER },
       { children: i18n.gettext('Extension'), value: ADDON_TYPE_EXTENSION },
+      { children: i18n.gettext('Search Tool'), value: ADDON_TYPE_OPENSEARCH },
```

Comment:
While "Themes" are much more popular/common than Search Tools, I'm with you and will stick with alphabetical order until UX yells at us 😉

Reply:
Yes :)